### ISSUE TITLE
Microsoft C++ Runtime Error

### ISSUE THREAD

**Issue by Keagin** [OpenShot/openshot-qt#1296]
**System Details:**
* Operating System / Distro: Windows 10
* OpenShot Version: V2.4.1
* _Please attach log files if crash_
![image](https://user-images.githubusercontent.com/36723495/36523223-3d8dc9fc-17a9-11e8-8c91-b935a43b1149.png)

**Issue Description and steps to reproduce:**
 I was adding pics and music and the system gave this error.

**Comment by peanutbutterandcrackers** [comment]
Whoa. That's one of a kind. A new low on Windows, it seems.

@Keagin - Thank you for the good issue report, good sir. And sorry for the inconvenience caused. Hopefully, this problem will be solved soon.

@DylanC - I think this one deserves your attention. :)

**Comment by peanutbutterandcrackers** [comment]
Wonder if this has to do more with libopenshot than openshot-qt...

**Comment by DylanC** [comment]
@peanutbutterandcrackers - Yeah its likely more to do with libopenshot. I might end up closing this one and opening a new one in the libopenshot project. 

What do you think?

**Comment by peanutbutterandcrackers** [comment]
@DylanC - Perhaps we should ask @Keagin to do so, since it is an issue brought to attention by him. And if he doesn't want to do so, then yes, maybe you should do it. :+1:

**Comment by blumarine0** [comment]
I have the same problem. How to fix it?

**Comment by peanutbutterandcrackers** [comment]
Perhaps trying out the latest [daily build](https://github.com/OpenShot/openshot-qt/releases/tag/daily) will help? Worth a shot, I think.

**Comment by Ramnic12** [comment]
Hello do we have a resolution or fix for this issue. I started getting the error after working on it for 4 days. Windows 10, Openshot 2.4.1. Any help will be appreciated. Thanks

**Comment by peanutbutterandcrackers** [comment]
@Ramnic12 - I really think using the latest [daily build](https://github.com/OpenShot/openshot-qt/releases/tag/daily) is worth a shot. It might have been fixed in the code by now. Please do try it out and see if it works for you. If not, we will get the devs attention on this issue, ASAP. Thanks!

**Comment by Ramnic12** [comment]
@peanutbutterandcrackers Thanks for the reply. I downloaded the build from daily build. version 2.4.1 -dev1. Uninstalled the corrupt exe, rebooted and reinstalled the above version. Error still showed. Rebooted the laptop again and this time no error. i was able to work on my projects. However the Transitioning feature is not working in this version. So I continued to work without using this feature as i had to get a vidio ready by 28th. . Launche.exe stops responding every-time transitioning is put in place.
